app/views/admin/facilities/show.html.erb

```diff
@@ -64,61 +64,60 @@
   </div>
 </div>
 
-<% if policy([:manage, :user, User]).index? %>
-  <div class="card">
-    <h2>Users</h2>
-    <% if @facility.users.present? %>
-      <% @facility.users.order(:full_name).each do |user| %>
-        <div class="card-row row">
-          <div class="col">
-            <h5>
-              <%= link_to user.full_name, [:admin, user] %>
-              <% unless user.sync_approval_status == 'allowed' %>
-                <span class="badge badge-primary <%= user.sync_approval_status %>"><%= user.sync_approval_status&.capitalize %></span>
-              <% end %>
-            </h5>
-            <div class="mt-1 mb-1">
-              <%=
-                if (user.logged_in_at.present?)
-                  'Last login: ' + rounded_time_ago_in_words(user.logged_in_at)
-                else
-                  'Has not logged in yet'
-                end
-              %>
-              <div class="text-info mt-2 mb-2">Access: <%= user.sync_approval_status_reason %></div>
-            </div>
-            <div>
-              <a href="tel:<%= user.phone_number %>" class="btn btn-sm btn-phone btn-outline-primary"><i class="fas fa-phone"></i> <%= user.phone_number %>
-              </a>
-            </div>
+<div class="card">
+  <h2>Users</h2>
+  <% if @facility_users.present? %>
+    <% @facility_users.order(:full_name).each do |user| %>
+      <div class="card-row row">
+        <div class="col">
+          <h5>
+            <%= link_to user.full_name, [:admin, user] %>
+            <% unless user.sync_approval_status == 'allowed' %>
+              <span class="badge badge-primary <%= user.sync_approval_status %>"><%= user.sync_approval_status&.capitalize %></span>
+            <% end %>
+          </h5>
+          <div class="mt-1 mb-1">
+            <%=
+              if (user.logged_in_at.present?)
+                'Last login: ' + rounded_time_ago_in_words(user.logged_in_at)
+              else
+                'Has not logged in yet'
+              end
+            %>
+            <div class="text-info mt-2 mb-2">Access: <%= user.sync_approval_status_reason %></div>
+          </div>
+          <div>
+            <a href="tel:<%= user.phone_number %>" class="btn btn-sm btn-phone btn-outline-primary"><i class="fas fa-phone"></i> <%= user.phone_number %>
+            </a>
           </div>
-          <div class="col text-right">
+        </div>
+        <div class="col text-right">
+          <% if current_admin.permissions_v2_enabled? %>
+            <%= link_to 'Edit', edit_admin_user_path(user), class: "btn btn-sm btn-outline-primary" %>
```

Comment:
Left out the `Delete` link. The old permissions implementation always returns `false` for `policy([:manage, :user, user]).destroy?`